src/renderer/src/views/components/Files/__tests__/index.test.ts

```
@@ -2,290 +2,181 @@ import { describe, it, expect, vi, beforeEach, afterEach } from 'vitest'
+    on: vi.fn((event: string, fn: Handler) => {
       if (!handlers.has(event)) handlers.set(event, new Set())
       handlers.get(event)!.add(fn)
     }),
-
-    off: vi.fn((event: string, fn: EventHandler) => {
+    off: vi.fn((event: string, fn: Handler) => {
       handlers.get(event)?.delete(fn)
     }),
-
     emit: vi.fn((event: string, ...args: any[]) => {
       handlers.get(event)?.forEach((fn) => fn(...args))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)